src/components/DataManager/Toolbar/ActionsButton.js

```diff
@@ -73,12 +74,18 @@ export const ActionsButton = injector(observer(({ store, size, hasSelected, ...r
         key={action.id}
         tag={Menu.Item}
         size={size}
-        onClick={(e) => {
+        // onClick={(e) => {
+        //   e.preventDefault();
+        //   e.stopPropagation();
+        //   action?.callback ? action?.callback(store.currentView?.selected?.selectedItems, action) : invokeAction(action, isDeleteAction);
+        //   parentRef?.current?.close?.();
+        // }}
+        onClick={useCallback((e) => {
```

Comment:
Inlining a hook like useCallback or useMemo like this can break with a minor change such as a conditional on the inclusion of this component which is rendering inline, or anything above it. Maybe just extract the declaration up with the other hooks for consistency, and visibility.

Reply:
done 😄